easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.5.1-GCC-12.2.0.eb

```
@@ -0,0 +1,55 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
+# Modified by: Adam Huffman <adam.huffman@crick.ac.uk>
+# The Francis Crick Institute
+# Modified by: Kurt Lust, UAntwerp
+# Modified by: Sebastien Moretti for non-x86_64 systems
+# SIB Swiss Institute of Bioinformatics
+# Modified by: Bente Barge <bente.s.barge@uit.no>
+
+name = 'Bowtie2'
+version = '2.5.1'
+_simde_version = '20220504'
+_simde_commit = 'cbef1c1'
+
+homepage = 'https://bowtie-bio.sourceforge.net/bowtie2/index.shtml'
+description = """ Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
+ to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
+ of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
+ Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
+ its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes."""
+
+toolchain = {'name': 'GCC', 'version': '12.2.0'}
+toolchainopts = {'pic': True, 'cstd': 'gnu++98'}
+
+source_urls = [('https://sourceforge.net/projects/bowtie-bio/files/%(namelower)s/%(version)s', 'download')]
+sources = ['%(namelower)s-%(version)s-source.zip']
+checksums = ['cb6cbbbb5a7167a2f21a3d63cb9774336361f540e1ec3d8ff907f955c35f71b8']
+
+# SIMD Everywhere implementations only, for non-x86_64 systems
+if ARCH != 'x86_64':
+    source_urls += ['https://github.com/simd-everywhere/simde-no-tests/archive']
+    sources += [
+        {'download_filename': '%s.zip' % _simde_commit,
+         'filename': 'simde-%s.zip' % _simde_version,
+         'extract_cmd': 'unzip %%s && mv simde*-%s*/* %%(namelower)s-%%(version)s/third_party/simde/' % _simde_commit,
+         }
+    ]
+    checksums += ['d01f084ef5ff69b0a9b96370ae314fe1e55ef3339b25afcd3385958ac0e6ad68']
```

updated checksum if using commit `6e9ac24`
```suggestion
    checksums += ['d961f5f2720c2148d88e687a5d5262d15b402f81675f62f217f60603466f8a4e']
```

---

Hi. First I committed this suggestion. Now having a second look, I think you meant to updated the actual checksum, not this "checksum +=" inside this IF-loop that is triggered for non-x86_64 systems? See my latest commit where I undid this change and updated the actual checksum.

---

no, I did mean to update the `simde` checksum
github made some changes to how they calculate checksums recently -- see [this comment](https://github.com/easybuilders/easybuild-easyconfigs/issues/5151#issuecomment-1410107202). The changes have since been reverted), so if you got a checksum error that was probably why

---

the `simde` checksum needs updating since the other suggestion bumped the version it downloads to a newer commit (which obviously has a different checksum)